README.md

```diff
@@ -1,33 +1,57 @@
-# Intel® nGraph™ Compiler and runtime for TensorFlow*
+# nGraph-Tensorflow : Intel® nGraph™ Compiler and Runtime for TensorFlow
 
```

Comment:
Yes, it would be nice to have a name for the repo. Earlier, we were referencing it as 'ngraph-tensorflow bridge'. Bridge is more of an internal usage, so we removed it. Our current version, does not explicitly call out a name but at many places we reference 'ngraph-tf bridge'. Requires more brainstorming.

Reply:
@sayantan-nervana That repo is now private - so won't matter much. I agree with @shresthamalik that it needs more brainstorming. For now I would recommend not introducing this.